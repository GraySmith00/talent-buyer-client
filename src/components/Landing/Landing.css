.landing {
  width: 100%;
  position: relative;
  background-color: #4d5261; }
  .landing .hero {
    display: flex;
    flex-direction: column;
    align-items: center;
    position: relative;
    width: 100%;
    background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("./savoy_ogden_optimized.jpg");
    background-size: cover;
    background-repeat: no-repeat;
    background-position: center center;
    -webkit-box-shadow: 0px 10px 103px -2px rgba(0, 0, 0, 0.75);
    -moz-box-shadow: 0px 10px 103px -2px rgba(0, 0, 0, 0.75);
    box-shadow: 0px 10px 103px -2px rgba(0, 0, 0, 0.75); }
  .landing h1 {
    margin: 0 auto;
    font-size: 6rem;
    margin-top: 7rem;
    color: #fff;
    font-family: 'Courgette', cursive; }
  .landing .divider {
    display: flex;
    justify-content: space-between;
    align-items: center;
    position: absolute;
    bottom: 1rem;
    width: 90%;
    font-size: 1.7rem;
    color: #fff; }
  .landing .divider-line {
    height: 2px;
    background-color: #fff;
    width: 45%; }
  .landing .market-dash {
    display: grid;
    grid-template-columns: repeat(2, 1fr);
    grid-gap: 1.5rem;
    padding: 7rem;
    color: #fff;
    background-color: #353945; }
  .landing .section-title {
    margin-top: 0;
    font-size: 2rem; }
  .landing .market-dash p {
    color: #b6b9c6; }
  .landing .market-dash img {
    -webkit-box-shadow: 8px 10px 76px -14px rgba(0, 0, 0, 0.75);
    -moz-box-shadow: 8px 10px 76px -14px rgba(0, 0, 0, 0.75);
    box-shadow: 8px 10px 76px -14px rgba(0, 0, 0, 0.75); }
