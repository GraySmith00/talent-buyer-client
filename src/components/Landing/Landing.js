import React, { Component } from 'react';
import SignUp from '../SignUp/SignUp';
import LogIn from '../LogIn/LogIn';

class Landing extends Component {
  render() {
    return (
      <div>
        <h1>Landing</h1>
        <SignUp />
        <LogIn />
      </div>
    );
  }
}

export default Landing;
