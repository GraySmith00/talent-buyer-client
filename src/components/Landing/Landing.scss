.landing {
  display: flex;
  flex-direction: column;
  align-items: center;

  height: 100%;
  width: 100%;
  position: relative;

  background-color: #131621;

  .hero {
    display: flex;
    justify-content: center;

    height: 60%;
    width: 100%;
    background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
      url('./savoy_ogden_optimized.jpg');
    background-size: cover;
    background-repeat: no-repeat;
    background-position: center center;
  }

  .button-container {
    text-align: center;
    position: absolute;
    background-color: #21222b;
    padding: 2rem 2rem;
    top: 50%;
    width: 30rem;
    border-radius: 10px;
    color: #fff;
  }

  h1 {
    margin: 0 auto;
    font-size: 6rem;
    margin-top: 7rem;
    color: #fff;
    font-family: 'Courgette', cursive;
  }

  .half-button {
    margin: 1em;
    width: 40%;
  }
}
