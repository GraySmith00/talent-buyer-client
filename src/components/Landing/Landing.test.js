import React from 'react';
import { shallow } from 'enzyme';

import Landing from './Landing';

describe('Landing component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Landing />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
