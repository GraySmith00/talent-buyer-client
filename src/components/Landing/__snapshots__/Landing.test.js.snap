// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Landing />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        false,
        false,
        <section
          className="hero"
>
          <h1>
                    Talent Buyer
          </h1>
          <withRouter(Splash)
                    openLogInModal={[Function]}
                    openSignUpModal={[Function]}
          />
          <section
                    className="divider"
          >
                    <div
                              className="divider-line"
                    />
                    <i
                              className="fas fa-angle-down"
                    />
                    <div
                              className="divider-line"
                    />
          </section>
</section>,
        <section
          className="market-dash"
>
          <div
                    className="market-dash-text"
          >
                    <h2
                              className="section-title"
                    >
                              Organize all your shows in the dashboard.
                    </h2>
                    <p>
                              Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
                    </p>
          </div>
          <img
                    alt="dashboard"
                    src="dashImage.jpg"
                    style={
                              Object {
                                        "height": "250px",
                                      }
                    }
          />
</section>,
        <section
          className="market-browse"
>
          <img
                    alt="dashboard"
                    src="dashImage.jpg"
                    style={
                              Object {
                                        "height": "320px",
                                      }
                    }
          />
          <div
                    className="market-dash-text"
          >
                    <h2
                              className="section-title"
                    >
                              Organize all your shows in the dashboard.
                    </h2>
                    <p>
                              Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
                    </p>
          </div>
</section>,
      ],
      "className": "landing",
    },
    "ref": null,
    "rendered": Array [
      false,
      false,
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Talent Buyer
</h1>,
            <withRouter(Splash)
              openLogInModal={[Function]}
              openSignUpModal={[Function]}
/>,
            <section
              className="divider"
>
              <div
                            className="divider-line"
              />
              <i
                            className="fas fa-angle-down"
              />
              <div
                            className="divider-line"
              />
</section>,
          ],
          "className": "hero",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Talent Buyer",
            },
            "ref": null,
            "rendered": "Talent Buyer",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "openLogInModal": [Function],
              "openSignUpModal": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="divider-line"
/>,
                <i
                  className="fas fa-angle-down"
/>,
                <div
                  className="divider-line"
/>,
              ],
              "className": "divider",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "divider-line",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "fas fa-angle-down",
                },
                "ref": null,
                "rendered": null,
                "type": "i",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "className": "divider-line",
                },
                "ref": null,
                "rendered": null,
                "type": "div",
              },
            ],
            "type": "section",
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="market-dash-text"
>
              <h2
                            className="section-title"
              >
                            Organize all your shows in the dashboard.
              </h2>
              <p>
                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
              </p>
</div>,
            <img
              alt="dashboard"
              src="dashImage.jpg"
              style={
                            Object {
                                          "height": "250px",
                                        }
              }
/>,
          ],
          "className": "market-dash",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2
                  className="section-title"
>
                  Organize all your shows in the dashboard.
</h2>,
                <p>
                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
</p>,
              ],
              "className": "market-dash-text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Organize all your shows in the dashboard.",
                  "className": "section-title",
                },
                "ref": null,
                "rendered": "Organize all your shows in the dashboard.",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                },
                "ref": null,
                "rendered": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "dashboard",
              "src": "dashImage.jpg",
              "style": Object {
                "height": "250px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": "section",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="dashboard"
              src="dashImage.jpg"
              style={
                            Object {
                                          "height": "320px",
                                        }
              }
/>,
            <div
              className="market-dash-text"
>
              <h2
                            className="section-title"
              >
                            Organize all your shows in the dashboard.
              </h2>
              <p>
                            Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
              </p>
</div>,
          ],
          "className": "market-browse",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "dashboard",
              "src": "dashImage.jpg",
              "style": Object {
                "height": "320px",
              },
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2
                  className="section-title"
>
                  Organize all your shows in the dashboard.
</h2>,
                <p>
                  Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
</p>,
              ],
              "className": "market-dash-text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Organize all your shows in the dashboard.",
                  "className": "section-title",
                },
                "ref": null,
                "rendered": "Organize all your shows in the dashboard.",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                },
                "ref": null,
                "rendered": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                "type": "p",
              },
            ],
            "type": "div",
          },
        ],
        "type": "section",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          false,
          false,
          <section
            className="hero"
>
            <h1>
                        Talent Buyer
            </h1>
            <withRouter(Splash)
                        openLogInModal={[Function]}
                        openSignUpModal={[Function]}
            />
            <section
                        className="divider"
            >
                        <div
                                    className="divider-line"
                        />
                        <i
                                    className="fas fa-angle-down"
                        />
                        <div
                                    className="divider-line"
                        />
            </section>
</section>,
          <section
            className="market-dash"
>
            <div
                        className="market-dash-text"
            >
                        <h2
                                    className="section-title"
                        >
                                    Organize all your shows in the dashboard.
                        </h2>
                        <p>
                                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
                        </p>
            </div>
            <img
                        alt="dashboard"
                        src="dashImage.jpg"
                        style={
                                    Object {
                                                "height": "250px",
                                              }
                        }
            />
</section>,
          <section
            className="market-browse"
>
            <img
                        alt="dashboard"
                        src="dashImage.jpg"
                        style={
                                    Object {
                                                "height": "320px",
                                              }
                        }
            />
            <div
                        className="market-dash-text"
            >
                        <h2
                                    className="section-title"
                        >
                                    Organize all your shows in the dashboard.
                        </h2>
                        <p>
                                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
                        </p>
            </div>
</section>,
        ],
        "className": "landing",
      },
      "ref": null,
      "rendered": Array [
        false,
        false,
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1>
                Talent Buyer
</h1>,
              <withRouter(Splash)
                openLogInModal={[Function]}
                openSignUpModal={[Function]}
/>,
              <section
                className="divider"
>
                <div
                                className="divider-line"
                />
                <i
                                className="fas fa-angle-down"
                />
                <div
                                className="divider-line"
                />
</section>,
            ],
            "className": "hero",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Talent Buyer",
              },
              "ref": null,
              "rendered": "Talent Buyer",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "openLogInModal": [Function],
                "openSignUpModal": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div
                    className="divider-line"
/>,
                  <i
                    className="fas fa-angle-down"
/>,
                  <div
                    className="divider-line"
/>,
                ],
                "className": "divider",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "divider-line",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "fas fa-angle-down",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "i",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "className": "divider-line",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "div",
                },
              ],
              "type": "section",
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="market-dash-text"
>
                <h2
                                className="section-title"
                >
                                Organize all your shows in the dashboard.
                </h2>
                <p>
                                Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
                </p>
</div>,
              <img
                alt="dashboard"
                src="dashImage.jpg"
                style={
                                Object {
                                                "height": "250px",
                                              }
                }
/>,
            ],
            "className": "market-dash",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="section-title"
>
                    Organize all your shows in the dashboard.
</h2>,
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
</p>,
                ],
                "className": "market-dash-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Organize all your shows in the dashboard.",
                    "className": "section-title",
                  },
                  "ref": null,
                  "rendered": "Organize all your shows in the dashboard.",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                  },
                  "ref": null,
                  "rendered": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "dashboard",
                "src": "dashImage.jpg",
                "style": Object {
                  "height": "250px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "section",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="dashboard"
                src="dashImage.jpg"
                style={
                                Object {
                                                "height": "320px",
                                              }
                }
/>,
              <div
                className="market-dash-text"
>
                <h2
                                className="section-title"
                >
                                Organize all your shows in the dashboard.
                </h2>
                <p>
                                Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
                </p>
</div>,
            ],
            "className": "market-browse",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "dashboard",
                "src": "dashImage.jpg",
                "style": Object {
                  "height": "320px",
                },
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="section-title"
>
                    Organize all your shows in the dashboard.
</h2>,
                  <p>
                    Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!
</p>,
                ],
                "className": "market-dash-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Organize all your shows in the dashboard.",
                    "className": "section-title",
                  },
                  "ref": null,
                  "rendered": "Organize all your shows in the dashboard.",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                  },
                  "ref": null,
                  "rendered": "Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos magni illum voluptates maiores eveniet minus eum laboriosam quos quo soluta!",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "section",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
